cypress/support/testdata/testdata_athena_reports.js

```diff
@@ -1,29 +1,8 @@
-export const athenaReportsTestDataStaging = {
-    test1: {
+export const athenaReportsTestDataStaging = [
+    {
         input: {
-            schools: [
-
-            ],
-            teachers: [
-                {
-                    //name is for UI filtering
-                    name: 'Tejal Admin',
-                    //id is for API filtering
-                    id: '362'
-                }
-
-            ],
-            runnables: [
-                {
-                    //name is for UI filtering
-                    name: 'Athena-Reports-AP-Test',
-                    //id is for API filtering
-                    id: 'externalactivity_1606'
-                }
-            ],
-            permissionForms: [
-
-            ],
+            teachers: ['Tejal Admin'],
```

Comment:
Its probably fine for the purposes of this PR, but we should plan to have a cautomationteacher10 or 11 user that has a more comprehensive test data for testing Athena reports; Something that covers:
- various interactives - unreportable, managed, generic
- runtimes - lara runtime vs activity player
- single vs multiple teachers in a class, 
- individual students vs working with collaborators,
- single vs multiple classes for the same student, 
- activity vs sequence, 
- different states of question interactives like answered and submitted, saved but not submitted, not attempted but ran the activity, not ran the activity